examples/jenkins/jenkins-persistent-template.json

```diff
@@ -204,11 +195,35 @@
       "value": "password"
     },
     {
+      "name": "MEMORY_REQUEST",
+      "displayName": "Memory Lower Limit",
+      "description": "Reserved amount of memory for the container.",
+      "value": "1200Mi"
+    },
+    {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory Upper Limit",
+      "description": "Maximum amount of memory the container can use.",
+      "value": "2000Mi"
+    },
```

Comment:
This should be just the upper limit. This memory would not be allocated unless needed. But I did not know about online forbidding setting requests. In that case which value do you propose? I am inclined to a higher value because OpenShift web console does not tell you that you container has been killed by OOM killer due to this setting (AFAIK). I have seen 2 defaults: 500Mi and 1000Mi. As I have tried different Jenkins containers, e.g. the modified one from docker.io/library/jenkins it has been killed by OOM killer (500Mi - every time, 1Gi - ~2) while initializing.